dis_snek/models/discord_objects/user.py

```diff
@@ -266,6 +266,22 @@ async def guild_permissions(self) -> Permissions:
 
         return permissions
 
+    async def has_permission(self, *permissions: Permissions) -> bool:
+        """
+        Checks if the member has all the given permission(s)
+        Args:
+            permissions: The permission(s) to check whether the user has it.
+        """
+
+        # Get the user's permissions
+        guild_permissions = self.guild_permissions()
```

Comment:
`self.guild_permissions()` is an async method. ~~I also think self.guild_permissions() method itself is broken 👀.~~

```suggestion
        guild_permissions = await self.guild_permissions()
```